scripts/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM ubuntu:latest AS expat-build
+
+ARG expat_version=2.6.3
+
+SHELL ["/bin/bash", "-c"]
+
+WORKDIR /workspace
+
+RUN apt update \
+    && apt install -y curl
+
+RUN curl -L https://github.com/libexpat/libexpat/releases/download/R_${expat_version//./_}/expat-${expat_version}.tar.gz | tar -xz \
+    && cd expat-${expat_version} \
+    && apt install -y build-essential \
+    && ./configure --prefix=/expat_lib \
+    && make && make install
+
+FROM ubuntu:latest
+
+COPY --from=expat-build /expat_lib /expat_lib
+
+RUN apt update && apt install -y curl libssl-dev libyaml-dev lua5.4 luarocks
```

Comment:
The reason of not merging the `apt install` into previous stage is to keep dependencies up-to-date when we change the expat version, isn't it?

Reply:
The tools installed in the expat-build stage are not in the final image. The reason I split it into two stages is that the expat-build stage has installed a lot of tools which we don't need in the final image.